tests/Fixer/PhpUnit/PhpUnitDedicateAssertFixerTest.php

```diff
@@ -251,6 +297,10 @@ public function testLegacyConfig()
         );
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation Option "functions" is deprecated and will be removed in 3.0, use option "target" instead.
```

Comment:
Haven't seen this annotation before. Doesn't seem to be something handled by PHPUnit, though (modifying does not affect test outcome).

Just a note for later?

Reply:
Symfony bridge.

```
ker@dus:~/github/PHP-CS-Fixer$ git du
diff --git a/tests/Fixer/PhpUnit/PhpUnitDedicateAssertFixerTest.php b/tests/Fixer/PhpUnit/PhpUnitDedicateAssertFixerTest.php
index 6a6756cb..8652446c 100644
--- a/tests/Fixer/PhpUnit/PhpUnitDedicateAssertFixerTest.php
+++ b/tests/Fixer/PhpUnit/PhpUnitDedicateAssertFixerTest.php
@@ -299,7 +299,7 @@ public function testLegacyConfig()
 
     /**
      * @group legacy
-     * @expectedDeprecation Option "functions" is deprecated and will be removed in 3.0, use option "target" instead.
+     * @expectedDeprecation Option "functions" is deprecated FOO and will be removed in 3.0, use option "target" instead.
      */
     public function testConfig()
     {
ker@dus:~/github/PHP-CS-Fixer$ vendor/bin/phpunit tests/Fixer/PhpUnit/PhpUnitDedicateAssertFixerTest.php 
PHPUnit 5.7.22 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.1.10-1+ubuntu17.04.1+deb.sury.org+1
Configuration: /home/keradus/github/PHP-CS-Fixer/phpunit.xml.dist

Testing PhpCsFixer\Tests\Fixer\PhpUnit\PhpUnitDedicateAssertFixerTest
.......................................................................................F.                                                                                                     89 / 89 (100%)

Time: 228 ms, Memory: 8.00MB

There was 1 failure:

1) PhpCsFixer\Tests\Fixer\PhpUnit\PhpUnitDedicateAssertFixerTest::testConfig
Failed asserting that format description matches text.
--- Expected
+++ Actual
@@ @@
 @expectedDeprecation:
-%A  Option "functions" is deprecated FOO and will be removed in 3.0, use option "target" instead.
+  Option "functions" is deprecated and will be removed in 3.0, use option "target" instead.

FAILURES!
Tests: 89, Assertions: 3556, Failures: 1.
```

Reply:
👍